src/Parser/MarksmanshipHunter/Modules/FocusChart/MaxFocus.js

```diff
@@ -0,0 +1,11 @@
+import Module from 'Parser/Core/Module';
+
+class MaxFocus extends Module {
+  on_byPlayer_cast(event){
+    if (event.sourceID === this.owner.player.id && (event.type === 'cast' || event.type === 'energize')){
```

Comment:
event.type can never be `enegerize` here since `on_byPlayer_cast` only gets `cast` events.